ui/packages/app/src/components/TimeSeries/TimeSeries.svelte

```diff
@@ -70,10 +70,22 @@
 		reader.readAsText(blob);
 	}
 
+	function dict_to_string(dict: any) {
```

Comment:
Can we add a better type for this. Looks like this should be something like:

```ts
interface TimeSeriesDict {
  data: Array<Array<unknown>>;
  headers: Array<string>;
}
```

Reply:
Actually, we already have some compatible types here: https://github.com/gradio-app/gradio/blob/cbaa090146010a41feec5338192c6992c517eeec/ui/packages/app/src/components/TimeSeries/TimeSeries.svelte#L25-L32

We might need to  add some conditional checks in the `dict_to_string` function but they should work otherwise.